service/controller/app/v1/resource/chartoperator/create.go

```diff
@@ -43,6 +45,21 @@ func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return nil
 	}
 
+	{
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding chart-operator deployment as helm v2 in %#q tenant cluster", release))
```

Comment:
```suggestion
		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding %#q deployment in tenant cluster", release))
```

I think message can be simplified and we can keep this change after the v2 to v3 migration.